Empire - The Last Battle/Assets/Unity/Scripts/MonoBehaviours/Army.cs

```diff
@@ -1,15 +1,18 @@
 ﻿using UnityEngine;
 using System.Collections.Generic;
 
-public class Army : MonoBehaviour {
```

Comment:
Try and stick to functions having { on the same line as the signature (you should be able to change visual studio to do this). Just keeps it consistant throughout.